staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -644,7 +667,10 @@ message LinuxContainerSecurityContext {
     // * localhost/<full-path-to-profile>: the profile installed on the node.
     //   <full-path-to-profile> is the full path of the profile.
     // Default: "", which is identical with unconfined.
-    string seccomp_profile_path = 10;
+    string seccomp_profile_path = 10 [deprecated=true];
+    // Seccomp profile for the contaner, defaults to the `RuntimeDefault`
```

Comment:
Can we defer the default to `RuntimeDefault` after we merge in the CRI changes?

Reply:
We can, I would prefer doing it now. We can work around in the runtimes and act accordingly to break no tests. :)